docs/jdbc-doris-catalog.md

```diff
@@ -0,0 +1,173 @@
+- Supports creating schema.
+- Supports dropping schema.
+
+### Schema properties
+
+- Support schema properties
+
+### Schema operations
+
+Refer to [Manage Relational Metadata Using Gravitino](./manage-relational-metadata-using-gravitino.md#schema-operations) for more details.
+
+## Table
+
+### Table capabilities
+
+- Gravitino's table concept corresponds to the Doris table.
+- Supports index.
+- Supports [column default value](./manage-relational-metadata-using-gravitino.md#table-column-default-value) and [auto-increment](./manage-relational-metadata-using-gravitino.md#table-column-auto-increment)..
+
+#### Table column types
+
+| Gravitino Type | Doris Type |
+|----------------|------------|
+| `Boolean`      | `Boolean`  |
+| `Byte`         | `TinyInt`  |
+| `Short`        | `SmallInt` |
+| `Integer`      | `Int`      |
+| `Long`         | `BigInt`   |
+| `Float`        | `Float`    |
+| `Double`       | `Double`   |
+| `Decimal`      | `Decimal`  |
+| `Date`         | `Date`     |
+| `Timestamp`    | `Datetime` |
+| `VarChar`      | `VarChar`  |
+| `FixedChar`    | `Char`     |
+| `String`       | `String`   |
+
+:::info
+MySQL doesn't support Gravitino `Fixed` `Struct` `List` `Map` `Timestamp_tz` `IntervalDay` `IntervalYear` `Union` `UUID` type.
+Meanwhile, the data types other than listed above are mapped to Gravitino **[Unparsed Type](./manage-relational-metadata-using-gravitino.md#unparsed-type)** that represents an unresolvable data type since 0.5.0.
+:::
+
+#### Table column auto-increment
+
+:::note
+Doris does not support auto-increment column before version 2.1.0
+So, Gravitino did not support auto-increment column before version 0.5.0
+:::
+
+
+### Table properties
+
+Doris supports table properties, and you can set them in the table properties.
+
+### Table indexes
+
+- Supports PRIMARY_KEY
+
+:::note
+Index can only apply to a single column
+:::
+
+<Tabs>
+<TabItem value="json" label="Json">
+
+```json
+{
+  "indexes": [
+    {
+      "indexType": "primary_key",
+      "name": "PRIMARY",
+      "fieldNames": [["id"]]
+    }
+  ]
+}
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+Index[] indexes = new Index[] {
+    Indexes.of(IndexType.PRIMARY_KEY, "PRIMARY", new String[][]{{"id"}})
+}
+```
+
+</TabItem>
+</Tabs>
+
+### Table operations
+
+Refer to [Manage Relational Metadata Using Gravitino](./manage-relational-metadata-using-gravitino.md#table-operations) for more details.
+
+#### Alter table operations
+
+Gravitino supports these table alteration operations:
+
+- `RenameTable`
+- `UpdateComment`
+- `AddColumn`
+- `DeleteColumn`
+- `UpdateColumnType`
+- `UpdateColumnPosition`
+- `UpdateColumnComment`
+- `SetProperty`
+
+:::info
+ - Not all table alteration operations can be processed in batch. 
+ - Schema change, such as add/modify/drop columns can be processed in batch
+ - Can modify multi column comments at same time
+ - You can't modify the column type and column comment at the same time
```

Comment:
Can you describe the asynchronous process of schema change here?

Reply:
done